kinesis-video-pic/src/utils/src/Mutex.c

```diff
@@ -284,7 +267,12 @@ STATUS defaultConditionVariableWait(CVAR cvar, MUTEX mutex, UINT64 timeout)
     INT32 retVal = 0;
     struct timespec timeSpec;
     pthread_mutex_t* pMutex = (pthread_mutex_t*) mutex;
+
+#if defined __linux__ || defined __CYGWIN__ || defined __MACH__
```

Comment:
Same question - perhaps we should have the same behavior for all of the platforms.

Reply:
It is same, since windows does not support pthread natively will  not run this portion of code

Reply:
Yeah, windows code is in the above section LN 115